README.md

```diff
@@ -110,7 +110,7 @@ timestamps {
 You can also install the `log-cli` plugin, and run:
 
 ```bash
-java -jar jenkins-cli.jar -s http://localhost:8080/jenkins/ tail-log io.jenkins.plugins.artifact_manager_s3 -l ALL
+java -jar jenkins-cli.jar -s http://localhost:8080/jenkins/ tail-log io.jenkins.plugins.artifact_manager_jclouds -l ALL
```

Comment:
Actually would be preferable to listen to both, but the plugin does not currently supported multiple arguments, and this package will produce by far the most messages.

Reply:
why not have packages `io.jenkins.plugins.artifact_manager_jclouds` and `io.jenkins.plugins.artifact_manager_jclouds.s3` that would allow listening to all messages?

Reply:
Yes, that would probably make sense. Too big a change for this PR.